### ISSUE TITLE
[development] symlinked module doesnt seem to be able to resolve the selected theme

### ISSUE THREAD

**Issue by swyxio** [ChristopherBiscardi/gatsby-theme#12]
so i have this repo cloned and yarn installed and i did `yarn workspace gatsby-theme link` and all.

it seems to work fine. however when i head to my test project (not in this repo; in a different repo) i get this error:

![image](https://user-images.githubusercontent.com/6764957/62440514-82086180-b71e-11e9-809c-d02673942dce.png)

dont take the line of the error literally as i've been trying to debug this.

basically it falls over here:

```js
const themeSrc = path.join(path.dirname(require.resolve(theme)), `src`);
```

and the resolving doesnt work thru the symlinking. is this expected? or is there something i should do? just trying to learn how to do this right.
